Userland/Applications/Spreadsheet/CMakeLists.txt

```diff
@@ -1,28 +1,36 @@
 compile_gml(CondFormatting.gml CondFormattingGML.h cond_fmt_gml)
 compile_gml(CondView.gml CondFormattingViewGML.h cond_fmt_view_gml)
+compile_gml(csv_import.gml CSVImportGML.h csv_import_gml)
+compile_gml(select_format_page.gml FormatSelectionPageGML.h select_format_page_gml)
 
 set(SOURCES
-    Cell.cpp
-    CellSyntaxHighlighter.cpp
-    CellType/Date.cpp
-    CellType/Format.cpp
-    CellType/Identity.cpp
-    CellType/Numeric.cpp
-    CellType/String.cpp
-    CellType/Type.cpp
-    CellTypeDialog.cpp
-    CondFormattingGML.h
-    CondFormattingViewGML.h
-    HelpWindow.cpp
-    JSIntegration.cpp
-    Readers/XSV.cpp
-    Spreadsheet.cpp
-    SpreadsheetModel.cpp
-    SpreadsheetView.cpp
-    SpreadsheetWidget.cpp
-    Workbook.cpp
-    main.cpp
-)
+        Cell.cpp
+        CellSyntaxHighlighter.cpp
+        CellType/Date.cpp
+        CellType/Format.cpp
+        CellType/Identity.cpp
+        CellType/Numeric.cpp
+        CellType/String.cpp
+        CellType/Type.cpp
+        CellTypeDialog.cpp
+        CondFormattingGML.h
+        CondFormattingViewGML.h
+        HelpWindow.cpp
+        ImportDialog.cpp
+        JSIntegration.cpp
+        Readers/XSV.cpp
+        Spreadsheet.cpp
+        SpreadsheetModel.cpp
+        SpreadsheetView.cpp
+        SpreadsheetWidget.cpp
+        Workbook.cpp
+        main.cpp
+        )
+
+set(GENERATED_SOURCES
+        CSVImportGML.h
+        FormatSelectionPageGML.h
+        )
```

Comment:
Mysterious reformatting here(?)

Reply:
CLion seems to have reformatted it on save.
I'll see if I can turn that off 🤔

Reply:
No Mo Mysterious Reformats (NMMR!)